server/services/meilisearch/config.js

```diff
@@ -155,12 +155,34 @@ module.exports = ({ strapi }) => {
       return contentTypeWithIndexName
     },
 
+    /**
+     * Remove sensitive fields (password, author, etc, ..) from entry.
+     *
+     * @param {object} options
+     * @param {Array<Object>} options.entries - The entries to sanitize
+     *
+     *
+     * @return {Array<Object>} - Entries
+     */
     removeSensitiveFields: function ({ entries }) {
       return entries.map(entry => {
         delete entry.createdBy
         delete entry.updatedBy
         return entry
       })
     },
+
+    /**
+     * Remove unpublished entries from array of entries.
+     *
+     * @param {object} options
+     * @param {Array<Object>} options.entries - The entries to filter.
+     *
+     *
+     * @return {Array<Object>} - Published entries.
+     */
+    removeUnpublishedArticles: function ({ entries }) {
+      return entries.filter(entry => !(entry.publishedAt === null))
```

Comment:
This can work? `return entries.filter(entry => !entry.publishedAt)`

Reply:
In your example I'm searching for a [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy). In my code, I'm specifically searching for a `null`. 
Strapi handles un-publishing by changing the value of the `publishedAt` field at `null`. I don't know how this field might get tampered with the only thing I know is that `null`  means unpublished. I prefer to only make this assumption with my condition.

Reply:
I agree if you meant: if this value is not `null` I want it to be treated as `true`, in order to have control of this behavior if this changes in the future. Otherwise, the lookup for falsy is totally applicable, don't you think?